_includes/js/queries.md

```diff
@@ -328,16 +336,58 @@ var fewWins = new Parse.Query("Player");
 fewWins.lessThan("wins", 5);
 
 var mainQuery = Parse.Query.or(lotsOfWins, fewWins);
-mainQuery.find({
-  success: function(results) {
-     // results contains a list of players that either have won a lot of games or won only a few games.
-  },
-  error: function(error) {
+mainQuery.find()
+  .then(function(results) {
+    // results contains a list of players that either have won a lot of games or won only a few games.
+  })
+  .catch(function(error) {
     // There was an error.
-  }
-});
+  });
 </code></pre>
 
-You can add additional constraints to the newly created `Parse.Query` that act as an 'and' operator.
 
-Note that we do not, however, support GeoPoint or non-filtering constraints (e.g. `near`, `withinGeoBox`, `limit`, `skip`, `ascending`/`descending`, `include`) in the subqueries of the compound query.
+### AND-ed query constraints
+
+If you want to find objects that match all conditions, you normally would use just one query. You can add additional constraints to the newly created `Parse.Query` that act as an 'and' operator.
+
+<pre><code class="javascript">
+var query = new Parse.Query("User");
+query.greaterThan("age", 18);
+query.greaterThan("friends", 0);
+query.find()
+  .then(function(results) {
+    // results contains a list of users both older than 18 and having friends.
+  })
+  .catch(function(error) {
+    // There was an error.
+  });
+</code></pre>
+
+Sometimes world is complexer than this simple example and you may need an compound query of sub queries. You can use `Parse.Query.and` method to construct a query that is an AND of the queries passed in. For instance if you want to find users in the age of 16 or 18 who have either no friends or at least 2 friends, you can do:
```

Comment:
I would switch complexer to more complex. I'm not entirely sure complexer is a word, but I could be wrong.

Additionally, I think the first and second sentence can be morphed together to explain that the simple query above may not be sufficient in all cases and Parse.Query.and solves the problem.

Here is an example of what this could look like:

For more complex queries, you can use the `Parse.Query.and` method to return a query that is an AND of the input queries. For instance if you want to find users at the age of 16 or 18 who have either no friends or at least 2 friends, you can do: